### PR TITLE
Update configure-exchange-server-for-hybrid-modern-authentication.md

### PR DESCRIPTION
**Pull request by Smartboss88**

### CODE REVIEW

File: microsoft-365/enterprise/configure-exchange-server-for-hybrid-modern-authentication.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ Requirements about linked mailboxes to be inserted.
 Since many prerequisites are common for both Skype for Business and Exchange, review [Hybrid Modern Authentication overview and prerequisites for using it with on-premises Skype for Business and Exchange servers](hybrid-modern-auth-overview.md). Do this  *before*  you begin any of the steps in this article.
 
 > [!NOTE]
-> Outlook Web App and Exchange Control Panel does not work with hybrid Modern Authentication.
+> Outlook Web App and Exchange Control Panel does not work with hybrid Modern Authentication.  In addition, publishing Outlook Web App and Exchange Control Panel through Azure AD Application Proxy is unsupported.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/configure-exchange-server-for-hybrid-modern-authentication.md]:
```suggestion
> Outlook Web App and Exchange Control Panel do not work with hybrid Modern Authentication.  In addition, publishing Outlook Web App and Exchange Control Panel through Azure AD Application Proxy is unsupported.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Smartboss88, kindly commit the additional suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

**Comment by Smartboss88** [comment]
Any update please? Sent from Mail for Windows From: Yogesh KumarSent: Thursday, September 22, 2022 1:46 AMTo: MicrosoftDocs/microsoft-365-docsCc: Smartboss88; MentionSubject: Re: [MicrosoftDocs/microsoft-365-docs] Update configure-exchange-server-for-hybrid-modern-authentication.md (PR ***@***.*** Please merge this PR. Thanks—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this P.  Thanks

**Pull request merged**